svgnative/CMakeLists.txt

```diff
@@ -64,10 +64,15 @@ CMAKE_DEPENDENT_OPTION(USE_CAIRO_EXAMPLE "Cairo example" TRUE "NOT LIB_ONLY;CAIR
 ################################
 if(CAIRO)
     find_package(PkgConfig)
-    include(FindCairo)
+    # include(FindCairo)
```

Comment:
Why is this commented out now? Can we remove it now? Does it require any system that wants to build with Cairo to use pkg-install? How is support on macOS?